app/models/search.rb

```diff
@@ -129,4 +130,9 @@ def is_private
     return nil unless parsed_query[:private].present?
     parsed_query[:private].first.downcase == "true"
   end
+
+  def muted
+    return false unless parsed_query[:muted].present?
+    parsed_query[:muted].first.try(:downcase) == "true"
+  end
```

Comment:
Can you add documentation for this prefix to https://github.com/octobox/octobox/blob/master/app/views/shared/_search_prefixes.html.erb#L60

Reply:
will do that